aten/src/ATen/native/cuda/group_norm_kernel.cu

```diff
@@ -29,8 +29,41 @@ constexpr int kCUDANumThreads = 256;
 constexpr int kReduceTileSize = 32;
 
 template <typename T>
-__global__ void RowwiseMomentsCUDAKernel(
-    int64_t N,
+__global__ void ApplyScaleBiasNHWCKernel(const T* X,
+  T* Y,
+  int64_t N,
+  int64_t H,
+  int64_t W,
+  int64_t C,
+  int64_t group_span,
+  acc_type<T, true>* a,
+  acc_type<T, true>* b) {
+  using T_ACC = acc_type<T, true>;
+  const int64_t i = blockIdx.x;
+  const int64_t tx = threadIdx.x;
+
+  for(int j = tx; j < group_span; j += blockDim.x) {
+    const int64_t index = i * group_span + j;
+    const int64_t cur_sample = index / (H*W*C);
+    int64_t remainder = index % (H*W*C);
+    const int64_t cur_h = remainder / (W*C);
+    remainder = remainder % (W * C);
+    const int64_t cur_w = remainder / (C);
+    const int64_t cur_channel = remainder % C;
+
+    auto cur_a = a[cur_sample * C + cur_channel];
+    auto cur_b = b[cur_sample * C + cur_channel];
+    Y[index] = (static_cast<T_ACC>(X[index]) + cur_b) * cur_a;
+  }
+}
```

Comment:
I'm actually not sure how to achieve this behavior using tensor iterators and `gpu_kernel` - so it may be the case that this is unneeded.